app/presentation/RoomItem/index.js

```diff
@@ -110,7 +110,7 @@ const RoomItem = React.memo(({
 							ellipsizeMode='tail'
 							numberOfLines={1}
 						>
-							{name}
+							{getName({ name, username })}
```

Comment:
You're only considering DMs.
If `Real Name` is set, all channel/groups/livechat will be `username` 🤷‍♂